client/src/pages/Onboarding.jsx

```diff
@@ -123,7 +124,9 @@ const Onboarding = (props) => {
   const options = ["JavaScript", "Python", "Java", "C++", "Ruby"];
   const levels = ["Beginner", "Intermediate", "Advanced"];
 
-  return (
+  return nextPage ? (
```

Comment:
```suggestion
  return nextPage ? (
  <Redirect from="/experience" exact to="/" />
  ) : (
```
No longer needed, This can be deleted.

Reply:
Done